pkg/bucketclass/bucketclass.go

```diff
@@ -39,14 +41,96 @@ func Cmd() *cobra.Command {
 // CmdCreate returns a CLI command
 func CmdCreate() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "create <bucket-class-name>",
+		Use:   "create",
 		Short: "Create bucket class",
+	}
+
+	cmd.AddCommand(
+		CmdCreateNamespaceBucketclass(),
+		CmdCreateRegularBucketClass(),
+	)
+	return cmd
+}
+
+// CmdCreateRegularBucketClass returns a CLI command
+func CmdCreateRegularBucketClass() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "regular <bucket-class-name>",
```

Comment:
not sure `regular` is a good name. 
maybe `placement-bucketclass` to be consisten with the CRD?

Reply:
@romayalon @dannyzaken 

So are we now defining "types" of bucket classes that the CLI creates?
I think the usage currently is a bit weird:
```
noobaa bucketclass create regular ...
noobaa bucketclass create namespace-bucketclass ...
```

For now, I think would suggest to keep a generic create command for bucket classes, and then define the policies to apply using flags such as --placementPolicy ... and --namespacePolicy ...
Actually getting to different "types" of bucketclasses might lead us to think about use cases more than policies, and then it will be like `noobaa bucketclass create hub-cache "my-cached-cloud-hub" --hub-connection ...` which will eventually set both placement policy and namespace policy.

Reply:
@guymguym I tried this command but it won't work because it's not one of the available commands (placement-bucketclass/namespace-buckeclass) actually I think that if I won't have this protection we might get into a situation of having both of the policies... 
I can change the naming to be namespace/placement policy if you feel it's better phrased.. WDYT?